libraries/engage/product/components/MediaSlider/components/MediaImage/index.jsx

```diff
@@ -14,13 +13,9 @@ const MediaImage = ({ media, onClick }) => (
     onKeyDown={onClick}
     role="button"
     tabIndex="0"
-    className={styles.full}
+    className={full}
   >
-    <ProductImage
-      src={media.url}
-      alt={media.altText}
-      resolutions={[PRODUCT_SLIDER_IMAGE_FORMATS[0]]}
-    />
+    <FeaturedImage url={media.url} altText={media.altText} hideImageShadow />
```

Comment:
Reusing the `FeaturedImage` will have side effects on the configuration. Nobody will expect the configuration of featured images to affect the PDP image.